Ubuntu-16.04/web-servers/lamp-phpmyadmin.yml

```diff
@@ -0,0 +1,40 @@
+#cloud-config
+apt_update: true
+package_upgrade: true
+packages:
+  - apache2
+  - apache2-utils
+  - php5-mysql
```

Comment:
That was not intended to be in this PR.  Still needs work and testing.  Removed.